build-script

```diff
@@ -6,7 +6,7 @@
 ## -------------------
 
 # Specify source and site branches
-SOURCE="master"
+SOURCE="main"
```

Comment:
I think `build-script` isn't used any more. IIRC it was the original way to build the site but was replaced by `Jenkinsfile`, and that does not call `build-script`. `Jenkinsfile` has similar shell fragments.

Only way to find out is delete `build-script` and see!

Can make this change and separately investigate the effect of `build-script`.

Reply:
There is no way `build-script` is active. It checks out the old default branch by name.

Reply:
Pushed a commit removing the `build-script` file. If no lights blink, and there's no smoke coming out anywhere, it should be safe to leave it out :laughing: but I agree it's probably not used. I thought it was used to build something locally only.

Reply:
Hazy memory but I think the first setup did not use the `Jenkinsfile` feature on Jenkins; instead it was a bash script and `build-script` is copy for reference.